src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -114,22 +118,36 @@ public async Task<ActionResult> UpdateOrganisationByIdAsync(Guid id, UpdateOrgan
         public async Task<ActionResult<CreateOrganisationResponseViewModel>> CreateOrganisationAsync(CreateOrganisationRequestViewModel viewModel)
         {
             if (viewModel is null)
-            {
                 throw new ArgumentNullException(nameof(viewModel));
-            }
 
-            //TODO: Task #6168
+            var result = await _createOrganisationService.CreateAsync(new CreateOrganisationRequest(
+                viewModel.OrganisationName,
+                viewModel.OdsCode,
+                viewModel.PrimaryRoleId,
+                viewModel.CatalogueAgreementSigned,
+                viewModel.Address is null ? null : new Address()
```

Comment:
The brackets `()` can be omitted when using object initialization.

Reply:
👍